cypress/integration/schema/model_spec.js

```diff
@@ -23,6 +23,22 @@ describe("Schema", () => {
 
     cy.get('button[kind="save"]').contains("Add Field").click();
 
+    cy.get('input[name="label"]').type(
+      "{leftarrow}{leftarrow}{leftarrow}{leftarrow}{leftarrow} inserting  text within {rightarrow} test",
+      {
+        force: true,
+      }
+    );
+
+    cy.get('input[name="name"]').type(
```

Comment:
For the test we want to check whether the `name` field has been set to the computer friendly version of the value entered into the `label` field.